### PR TITLE
Adding info on building the Template Validator

### PR DESCRIPTION
**Pull request by borod108**
Adding a little bit to the readme on how to build and push the Template
Validator.

**Release note**:
```
NONE
```

Signed-off-by: borod108 <boris.od@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,7 +61,30 @@ manifests and the operator can be deployed using:
...
+```
+
+The repository and image name and tag can be changed
+with these variables:
+```shell
+export VALIDATOR_REPOSITORY=<registry>/<image_name> # for example: export VALIDATOR_REPOSITORY=quay.io/kubevirt/kubevirt-template-validator
+export VALIDATOR_IMG_TAG=<image_tag> # for example: export VALIDATOR_IMG_TAG=latest
+```
+
+You should also edit the deployment ot pull the image you want, for example you can use this command:
```
Comment (ksimon1) [inline README.md:84]: typo ot > to

Comment (borod108) [reply README.md]: thank you for the catch.

**Pull request merged**